### PR TITLE
Migrate account-fiscal-rule to V10

### PR DESCRIPTION
**Pull request by yk2kus**

### CODE REVIEW

File: account_fiscal_position_rule/__manifest__.py [hunk 1]
```diff
@@ -8,9 +8,9 @@
 
 {
     'name': 'Account Fiscal Position Rule',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Generic Modules/Accounting',
-    'author': "Akretion,Odoo Community Association (OCA)",
+    'author': "AkretionOdoo Community Association (OCA)",
```
Comment (rvalyi) [inline account_fiscal_position_rule/__manifest__.py]: please put back Akretion, Odoo Community Association (OCA)

**Comment by rvalyi** [comment]
:+1:  thanks @yk2kus !

**Comment by mileo** [comment]
:+1:

**Comment by yk2kus** [comment]
Thanks for the help and guidance @rvalyi @mileo.

**Comment by renatonlima** [comment]
:+1:

**Pull request merged**